src/toxic_constants.h

```diff
@@ -23,6 +23,8 @@
 #ifndef TOXIC_CONSTANTS_H
 #define TOXIC_CONSTANTS_H
 
+#include <stdio.h>  // needed for FILE
+
```

Comment:
<stdbool.h> for bool and <stdint.h> for uint{8,16}_t as well.

Reply:
Done.